# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-byte-streams/respond-after-enqueue.any.js.ini
# Commit: c34fddbfd51b
# Full Hash: c34fddbfd51b86477ee9637f291228d8e6c5f939
# Author: Alexandra Borovova <aborovova@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - Mark all tests running in shadowrealm-in-serviceworker as intermittently failing or timing out.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230557
# ==============================================================================

diff -r 2afdda37f63b -r c34fddbfd51b testing/web-platform/meta/streams/readable-byte-streams/respond-after-enqueue.any.js.ini
--- a/testing/web-platform/meta/streams/readable-byte-streams/respond-after-enqueue.any.js.ini	Thu Nov 28 15:09:12 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-byte-streams/respond-after-enqueue.any.js.ini	Thu Nov 28 16:46:27 2024 +0000
@@ -30,11 +30,7 @@
     ERROR
 
 [respond-after-enqueue.https.any.shadowrealm-in-serviceworker.html]
-  expected:
-    if sessionHistoryInParent and debug and (os == "android"): ERROR
-    if sessionHistoryInParent and not debug and (os == "win"): ERROR
-    if sessionHistoryInParent and not debug and (os == "mac"): ERROR
-    [ERROR, TIMEOUT]
+  expected: [ERROR, TIMEOUT]
 
 [respond-after-enqueue.any.shadowrealm-in-dedicatedworker.html]
   expected: ERROR